src/ffi.rs

```diff
@@ -48,22 +49,29 @@ pub unsafe extern "C" fn install(
     bundle: *const c_char,
     vendor: *const c_char,
     name: *const c_char,
-    exec: *const c_char,
+    exec_args: *const *const c_char,
```

Comment:
Just wondering, why are we making this change? Wouldn't it be simpler to leave it as a single string parameter rather than an array and a length?

Reply:
@m-cat, this is required to to solve the issue with whitespace in paths (e.g. `C:\Program files\SAFE Browser`), which isn't handled correctly by some OSes, as it can't be distinguished from command-line arguments. And this interface actually goes in accordance with many system-level APIs. E.g. POSIX `exec` takes arguments as an array too rather than as a single string parameter.

Reply:
Never mind, just read the explanation for this on Slack.